mozilla/fx-private-relay
# MPP-3260: Updated the info icon tool-tip

This PR fixes 3260.

How to test:




# New feature description

In a desktop view, when clicking the info icon next to your relay domain on the accounts/profile page, a tool-tip appears instead of a modal. When you switch to a mobile view, a modal appears instead of the tool-tip. 


# Screenshot (if applicable)

Desktop:

![image](https://github.com/mozilla/fx-private-relay/assets/59676643/f98d5da1-3174-405b-bf6b-4059548f066d)

Mobile:

![image](https://github.com/mozilla/fx-private-relay/assets/59676643/a8b8a7b5-65e4-4ebe-a40c-e1da9eca8c71)


# How to test

Use different viewports to check whether the tooltip is responsive. When switching to a mobile viewport and clicking the info icon, a modal should appear instead of the tooltip.


# Checklist (Definition of Done)
